massa-node/src/main.rs

```diff
@@ -149,8 +159,25 @@ async fn launch(
     let stop_signal = signal::ctrl_c();
     tokio::pin!(stop_signal);
 
+    // info!("{:?}", SETTINGS.bootstrap.bootstrap_list);
+
+    let bootstrap_list = match boot {
+        true =>  {
+            let x = load_file2();
```

Comment:
You are not using the return of the function that read the file you are just using the number of entries in it and you are generating new public key instead of using the ones in the file.